### PR TITLE
MGMT-13107: Create signature to detect MTU mismatch betwen interface and cluster network

### PR DESCRIPTION
**Pull request by ori-amizur**
In case of multi node cluster that uses SDN network type, there might be a situation that the cluster network uses jumbo frame MTU, and there is a host that doesn't use jumbo frames.  In this case SDN fails to initialize and the installation fails.
Original triaging ticket: https://issues.redhat.com/browse/AITRIAGE-4847

/cc @filanov 
/cc @omertuc

### CODE REVIEW

File: tools/add_triage_signature.py [hunk 1]
```diff
@@ -1943,6 +1943,30 @@ def _get_interfaces(self, host: dict):
...
 
+class NetworksMtuMismatch(ErrorSignature):
+    LOG_PATTERN = re.compile(r'Failed to start sdn: interface MTU [(]([0-9]+)[)]'
+                             r' is too small for specified overlay MTU [(]([0-9]+)[)]')
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            comment_identifying_string="h1. Networks MTU Mismatch",
```
Comment (omertuc) [inline tools/add_triage_signature.py:1954]: `ErrorSignature`s should also set `function_impact_label`

File: tools/add_triage_signature.py [hunk 2]
```diff
@@ -1943,6 +1943,30 @@ def _get_interfaces(self, host: dict):
...
+        try:
+            sdn_logs = get_triage_logs_tar(
+                triage_url=url, cluster_id=get_metadata_json(url)["cluster"]["id"]
+            ).get("controller_logs.tar.gz/must-gather.tar.gz/must-gather.local.*/*/namespaces/openshift-sdn/pods/sdn-*/sdn/sdn/logs/*.log")
+        except FileNotFoundError:
+            return
+        match = self.LOG_PATTERN.search(sdn_logs)
+        if match:
+            self._update_triaging_ticket("SDN failed to start: Overlay (cluster) network MTU {}"
+                                         " is bigger than the interface MTU {}".format(match[2], match[1]))
```
Comment (omertuc) [inline tools/add_triage_signature.py:1967]: nit: can be simplified

```suggestion
        if match := self.LOG_PATTERN.search(sdn_logs):
            self._update_triaging_ticket(f"SDN failed to start: Overlay (cluster) network MTU {match[2]}"
                                         " is bigger than the interface MTU {match[1]}")
```

Comment (ori-amizur) [reply tools/add_triage_signature.py:1967]: OK

**Review by osherdp — APPROVED** [review]

**Pull request merged**